_translator-files/po/de/Installation-for-Linux.po

```diff
@@ -42,7 +43,7 @@ msgstr "Installation auf Linux"
 #. type: Plain text
 #: ../wiki/en/Installation-for-Linux.md:9
 msgid "{% include breadcrumb.html root=\"Using Jamulus\" branch1=\"Getting Started\" branch1-url=\"Getting-Started\" %}"
-msgstr "{% include breadcrumb.html root=\"Jamulus Nutzen\" branch1=\"Erste Schritte\" branch1-url=\"Getting-Started\" %}"
+msgstr "{% include breadcrumb.html root=\"Jamulus verwenden \" branch1=\"Setup - Erste Schritte mit Jamulus\“ branch1-url=\"Getting-Started\" %}"
```

Comment:
When you open this file in Poedit it should display a message: "Invalid control sequence". There's a backslash that shouldn't be there (see source text in Poedit). My bad, I thought these backslashes were visible in the editor but they're not, so no need to manually add them.

Reply:
In Installation for IoS the \ is accepted without errors

<img width="1032" alt="grafik" src="https://user-images.githubusercontent.com/77500932/168583393-bd588789-194d-4c54-b8b5-13b60169548f.png">